hack/.shellcheck_failures

```diff
@@ -123,7 +123,6 @@
 ./test/images/volume/rbd/bootstrap.sh
 ./test/images/volume/rbd/create_block.sh
 ./test/images/volume/rbd/mon.sh
```

Comment:
this file should come out too, shouldn't it?

Reply:
@cblecker the lint should catch which files need to come out and fail if they can come out

Reply:
So we've fixed one shellcheck issue, but not all of them in this file?

Reply:
Each `.sh` file in this file has shellcheck problems and they are being fixed.

Reply:
Ref ：https://github.com/kubernetes/kubernetes/issues/68026
>Step 1: Remove one or two packages (related packages) from the list in -

Reply:
you're making changes to `test/images/volume/rbd/mon.sh`, but this file is still listed in the failures file

Reply:
Sorry is I forget to delete `test/images/volume/rbd/mon.sh` so check passed. I'll get right on it. I'll fix it